frontend/src/core/app/helpers.js

```diff
@@ -238,7 +238,70 @@ define(function(require){
           } else {
             return block.inverse(this);
           }
+        },
+
+        validateCourseContent: function() {
+          // Store current course
+          var currentCourse = Origin.editor.data.course;
```

Comment:
I wonder whether the course model should be injected as an argument to this function instead of relying on Origin? That would allow this function to have tests written for it and would allow additional usage in a non-course context.